code/modules/fish/fishtank.dm

```diff
@@ -522,6 +522,8 @@
 		return ..()
 
 /obj/machinery/fishtank/attack_hand(mob/user)
+	if(isAI(user)) //No tapping the fish tanks for you!
```

Comment:
would be better to see what other mobs can call `attack_hand`.

This may be better to just prevent anyone who's not human (or at least carbon) from bonking on it.

Reply:
oh I made it like this to let silicons tap the glass. Since they kinda can do it using their bits.

Reply:
https://github.com/ParadiseSS13/Paradise/blob/master/code/game/objects/structures/crates_lockers/closets.dm#L314-L316 Might offer a good way to handle this. Utilize the attack_ai proc and call attack_hand if the mob is a borg, or toggle the light if AI?